### ISSUE TITLE
Wrong command for dropping Materialized Views 

### ISSUE THREAD

**Issue by ernestoongaro** [databricks/dbt-databricks#579]
### Describe the bug
When switching from   `materialized='materialized_view'` to   `materialized='materialized_view'` an incorrect statement is issued

```
11:40:17 Dropping relation `some`.`dbt_eongaro`.`test` because it is of type materializedview
11:40:17 On model.umc_poc.test: drop materializedview if exists `some`.`dbt_eongaro`.`test`
11:40:17 Databricks adapter: Error while running:
drop materializedview if exists `some`.`dbt_eongaro`.`test`
```

### Steps To Reproduce
1. Create a new model with  `materialized='materialized_view'
2. dbt run
3. change model to `materialized='view'
4. dbt run
5. See error ```drop materializedview if exists `some`.`dbt_eongaro`.`test```

### System information
```
12:02:24 Running with dbt=1.7.7
12:02:24 Registered adapter: databricks=1.7.3
```

**Comment by benc-db** [comment]
Sorry for the delay, the beta with the fix is out today (1.8.0b1).

**Comment by benc-db** [comment]
https://github.com/databricks/dbt-databricks/discussions/595
